src/component/menu/drawer.jsx

```diff
@@ -29,7 +29,7 @@ function getIcon(name) {
             />
         );
     } else {
-        return <Icon className={styles.navigationIcon}>{name}</Icon>;
+        return <IconComponent className={styles.navigationIcon} />;
```

Comment:
@FredrikOseberg Can you help me point to the API or the correct `library_icons` handling. I couldn't find a class like Github. I might be missing something here